docs/core_modules/model_modules/embeddings/modules.md

```diff
@@ -8,6 +8,8 @@ maxdepth: 1
 ---
 /examples/embeddings/OpenAI.ipynb
 /examples/embeddings/Langchain.ipynb
+/examples/embeddings/gradient_base_model.ipynb
+/examples/embeddings/gradient_model_adapter.ipynb
```

Comment:
the embeddings flow shouldn't really change based on us using a model adapter or base model, let's just provide one example

Reply:
thought about that later, didn't update the paths. fixed.